File: 242-valid-anagram/242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        
        
        dict = defaultdict(int)
        for x in s:
            dict[x] +=1 
        for y in t:
            dict[y] -=1
        
        return all( x==0 for x in dict.values())

File: 242-valid-anagram/242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        

        if(s.length() != t.length()) return false ; 

        int[] cntr = new int[26] ; 
        for(int i=0; i< s.length(); i++){
            cntr[s.charAt(i)-'a'] ++; 
            cntr[t.charAt(i)-'a'] --;
        }
        for(int c:cntr){
            if(c!=0){
                return false ; 
            }}
        return true ; 


        
    }
}